### PR TITLE
Don't duplicate writing guide, instead point to existing writing guide

### PR DESCRIPTION
**Pull request by rschu1ze**
Also see https://github.com/ClickHouse/clickhouse-docs/pull/694

### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

**Review by SmitaRKulkarni — APPROVED** [review]
LGTM

**Comment by rschu1ze** [comment]
@SmitaRKulkarni Thanks for checking!

(just to mention it: docs PRs currently need no *mandatory* review/approval but it is of course nice to have another pair of eyes on them 😄 )

**Pull request merged**
